docs/guide/design/icon-design-guide.md

```diff
@@ -6,58 +6,111 @@ title: Icon Design Guide
 
 Here are rules that should be followed to keep quality and consistency when making icons for Lucide.
 
-## Summary of the rules we have
-
-1. Icons must be designed on a **24 by 24 pixels** canvas.
-2. Icons must have at least **1 pixel padding** within the canvas.
-3. Icons must have a **stroke width of 2 pixels**.
-4. Icons must use **round joins**.
-5. Icons must use **round caps**.
-6. Icons must use **centered strokes**.
-7. Shapes (such as rectangles) in icons must have **border radius of 2 pixels**.
-8. Distinct elements must have **2 pixels of spacing between each other**.
-
-## The Rules Visualized
-
-### 1. Icons must be designed on a 24 by 24 pixels canvas.
+### 1. Icons must be designed on a **24 by 24 pixels** canvas.
 
 ![24px-24px](../../images/24px-24px.svg?raw=true "24px-24px")
 
-### 2. Icons must have at least 1 pixel padding within the canvas.
+## 2. Icons must have at least **1 pixel padding** within the canvas.
 
 ![1px-padding](../../images/1px-padding.svg?raw=true "1px-padding")
 
-### 3. Icons must have a stroke width of 2 pixels.
+## 3. Icons must have a **stroke width of 2 pixels**.
 
 ![2px-stroke](../../images/2px-stroke.svg?raw=true "2px-stroke")
 
-### 4. Icons must use round joins.
+## 4. Icons must use **round joins**.
 
 ![round-joints](../../images/round-joints.svg?raw=true "round-joints")
 
-### 5. Icons must use round caps.
+## 5. Icons must use **round caps**.
 
 ![round-caps](../../images/round-caps.svg?raw=true "round-caps")
 
-### 6. Icons must use centered strokes.
+## 6. Icons must use **centered strokes**.
 
 ![centered-strokes](../../images/centered-strokes.svg?raw=true "centered-strokes")
 
-### 7. Shapes (such as squares) in icons must have border radius of 2 pixels.
+## 7. Shapes (such as rectangles) must have a **border radius of**
+
+### C. **2 pixels** if they are at least 8 pixels in size
 
 ![2px-border-radius](../../images/2px-border-radius.svg?raw=true "2px-border-radius")
 
-### 8. Distinct elements must have 2 pixels of spacing between each other.
+### B. **1 pixel** if they are smaller than 8 pixels in size
+
+![1px-border-radius](../../images/1px-border-radius.svg?raw=true "1px-border-radius")
+
+## 8. Distinct elements must have **2 pixels of spacing between each other**
```

Comment:
Would be cool to also somehow mention this.

https://github.com/lucide-icons/lucide/pull/1224#issuecomment-1537442393